config/manager/network_policy.yaml

```diff
@@ -0,0 +1,21 @@
+---
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: allow-ingress-https-only-and-block-egress
+  namespace: system
+spec:
+  podSelector:
+    matchLabels:
+      control-plane: controller-manager
+  policyTypes:
+  - Ingress
+  - Egress
+  ingress:
+  - ports:
+    - protocol: TCP
+      port: 443,444,8080,8443,4443,9443
+      tls:
+        termination: passthrough
+  egress: []
+---
```

Comment:
```suggestion
```
Remove this line and make sure you have a newline at the end (notice the red symbol).